ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -393,6 +396,10 @@ nnfw_tensor_type_from_gst (const tensor_type type, NNFW_TYPE * nnfw_type)
     case _NNS_INT32:
       *nnfw_type = NNFW_TYPE_TENSOR_INT32;
       break;
+    case _NNS_UINT8:
+      /** @todo: update this to NNFW_TYPE_TENSOR_UINT8 type once nnfw is updated */
+      *nnfw_type = NNFW_TYPE_TENSOR_QUANT8_ASYMM;
```

Comment:
When they update QUANT8_ASYMM to UINT8 in the future, they will get build errors from OBS/Dashboard. They will be required to do "Group-SR" at that time. Simply be notified for this issue; you will be required to do group-SR with nnfw developers at the same time.

Reply:
I will do Group-SR.
Note: They will not update QUANT8_ASYMM to UINT8. They will add an extra data type UINT8.